project/AppDependencies.scala

```diff
@@ -20,9 +20,10 @@ object AppDependencies {
     "com.typesafe.play" %% "play-json-joda" % "2.6.10"
   )
 
-  val test = Seq(
+  val test: Seq[ModuleID] = Seq(
     //TODO remove hmrctest dependency
     "uk.gov.hmrc" %% "hmrctest" % "3.8.0-play-26" % "test",
+    "com.github.tomakehurst" % "wiremock" % "2.24.1" % "test",
```

Comment:
There is something like `wiremock-jre8` would allow you to not have all overrides

Reply:
I'll give it a try

Reply:
Works, nice 🥇